lib/auth/webauthn/login_passwordless.go

```diff
@@ -63,7 +63,10 @@ func (f *PasswordlessFlow) Begin(ctx context.Context) (*wantypes.CredentialAsser
 		Scope:      mfav1.ChallengeScope_CHALLENGE_SCOPE_PASSWORDLESS_LOGIN,
 		AllowReuse: mfav1.ChallengeAllowReuse_CHALLENGE_ALLOW_REUSE_NO,
 	}
-	return lf.begin(ctx, "" /* user */, chalExt, nil)
+	return lf.begin(ctx, BeginParams{
+		User:                "", /* user */
```

Comment:
We don't need the comment anymore, the field name already says what it is. That's the beauty of struct initialization.

```suggestion
		User:                "",
```